rmgpy/rmg/react.py

```diff
@@ -93,7 +93,8 @@ def react_all(core_spc_list, numOldCoreSpecies, unimolecularReact, bimolecularRe
     reactions and splits reaction families per task for improved load balancing in parallel runs.
     """
 
-    families = getDB('kinetics').families.keys()
+    family_names = getDB('kinetics').families.keys()
```

Comment:
You can avoid calling `getDB` twice by doing the following:

```
family_names, families = list(getDB('kinetics').families.items())
```

Again, `list` for python3 reasons

Reply:
Sorry I realized that I made a mistake here. There is a way to do it in python 3 I think but it might be more trouble than it is worth. Let's just revert this back, you had it right the first time

Reply:
Updated and tested to work in python 3.